.github/workflows/ci.yaml

```diff
@@ -35,14 +35,14 @@ jobs:
               npx lerna exec "yarn build --target i686-pc-windows-msvc" --concurrency 1 --stream --no-prefix
               yarn test
             target: 'i686-pc-windows-msvc'
-          - host: ubuntu-latest
+          - host: macos-latest
             architecture: x64
             target: 'x86_64-unknown-linux-gnu'
-            docker: |
-              docker pull $DOCKER_REGISTRY_URL/napi-rs/napi-rs/nodejs-rust:lts-debian
-              docker tag $DOCKER_REGISTRY_URL/napi-rs/napi-rs/nodejs-rust:lts-debian builder
+            setup: |
+              yarn global add lerna --registry https://registry.npmjs.org --network-timeout 300000
             build: |
-              docker run --rm -v ~/.cargo/git:/root/.cargo/git -v ~/.cargo/registry:/root/.cargo/registry -v $(pwd):/build -w /build builder yarn build
+              lerna exec "yarn build --target x86_64-unknown-linux-gnu --zig --zig-abi-suffix 2.17" --concurrency 1 --stream --no-prefix
+              sh -c '$(brew --prefix llvm)/bin/llvm-strip packages/*/*.node'
```

Comment:
https://github.com/napi-rs/napi-rs/pull/1026 adds a `--strip` option so you don't need to install llvm only for stripping symbols.